core/src/main/scala/com/stratio/crossdata/launcher/SparkJobLauncher.scala

```diff
@@ -104,6 +126,8 @@ object SparkJobLauncher extends SparkLoggerComponent with CrossdataSerializer {
       .addAppArgs(appArgs: _*)
       .setMaster(master)
       .setDeployMode("cluster")
+    // TODO startApplication(listener) since 1.6 preferred
+    sparkLauncher.launch()
```

Comment:
The process is started here but in the next lines we're still setting configuration parameters... is this correct?